### DISCUSSION TITLE
Documentation for DelayedJob Backgrounding

### DISCUSSION THREAD

**Discussion by jeremylynch** [shrinerb/shrine#621] [General]
The existing [background documentation](https://shrinerb.com/docs/plugins/backgrounding) focuses on Sidekiq, and this setup is not functional for delayed Job. 

Does anyone have any guidance for setting up DelayedJob and Shrine? 

```
Shrine::Attacher.promote_block do
  Delayed::Job.enqueue PromoteJob.new(self.class.name, record.class.name, record.id, name, file_data), queue: 'photos'
end
```

**Comment by janko** [comment]
There is a [wiki page](https://github.com/shrinerb/shrine/wiki/Backgrounding-Libraries) showing various libraries, but I see now that Delayed Job is missing. Doesn't the code you provided work?
